odoo/addons/base/static/src/scss/onboarding.scss

```diff
@@ -1,409 +1,164 @@
-        //transition: transform $o-onboarding-dot-timing;
-        box-shadow: 0 2px 10px gray(600);
-        padding: $o-horizontal-padding;
-        background-color: white;
-        color: $o-main-text-color;
-        opacity: 0;
-        z-index: -1;
-        &.o_onboarding_steps_just_done {
-            animation: o_onb_fadeIn ($o-onboarding-dot-timing*3);
-        }
-        &.o_onboarding_steps_done {
-            opacity: 1;
-            z-index: auto;
-        }
-    }
-
-    .o_onboarding_steps, .o_onboarding_step {
-        display: flex;
-    }
-
-    .o_onboarding_steps {
-        flex-flow: row nowrap;
-        justify-content: initial;
+        box-shadow: 0 -7px 20px -5px rgba(#000, 0.3);
+        animation: o_onboarding_slideInUp $o-onboarding-base-time ease-out;
     }
 
+    // = Step Design
+    // ------------------------------------------------------------------------
     .o_onboarding_step {
-        flex-flow: column nowrap;
-        flex: 1 1 auto;
-        align-items: center;
-        text-align: center;
-        position: relative;
-        opacity: 0.6;
-
-        @include media-breakpoint-down(sm) {
-            flex: 1 0 auto;
-            margin-bottom: 16px;
-            align-items: baseline;
-
-            .o_onboarding_step_content {
-                max-width: none;
-            }
-        }
-
-        &:first-of-type {
-            .o_onboarding_progress {
-                display: none !important;
-            }
+        flex: 1 1 0;
 
-            .o_onboarding_dot {
-                transition-delay: 0s;
+        @for $i from 1 through 5 {
+            &.o_onboarding_step__todo:nth-child(#{$i}) {
+                @include print-variable('o-onboarding-animation-delay', $i * .15s);
             }
         }
 
         .o_onboarding_step_content {
-            @include media-breakpoint-up(lg) {
-                min-width: 230px;
-            }
-            /* Progressbar should not break in mobile view */
-            @include media-breakpoint-down(sm) {
-                min-width: 230px;
-            }
-            max-width: 230px;
-            padding: 0 10px;
-
-            .o_onboarding_step_action {
-                border: 1px solid gray('200');
-                display: none;
-            }
+            width: $o-onboarding-step-width;
+        }
 
-            p {
-                opacity: 0.8;
-            }
+        .o_onboarding_step_action {
+            min-width: $o-onboarding-step-width * .5;
         }
 
         .o_onboarding_progress {
-            @include o-position-absolute($o-onboarding-dot-size*0.5 - 2px, calc(50% + 10px));
-            width: calc(100% - 20px);
-            height: 4px;
-            background-color: gray('200');
-
-            .progress-bar {
-                background-image: linear-gradient(90deg, $o-brand-primary, $o-brand-odoo);
-                transition: opacity $o-onboarding-dot-timing;
-            }
+            height: 2px;
```

Comment:
Should we define a variable for that?

Reply:
Done 👍 
Not strictly necessary but more readable, agreed